src/main/java/com/hedvig/backoffice/services/claims/ClaimsServiceClientFallback.java

```
@@ -71,12 +71,6 @@ public Claim find(String id, String token) {
         return null;
     }
 
-    @Override
-    public List<ClaimType> types(String token) {
-        log.error("request to claim-service failed");
-        return new ArrayList<>();
-    }
```

Why does this class even exist? Didn't know we had a "Fallback" class

---

Yeah I was surprised as well. I guess it does what it's supposed to, but every method in this fallback just logs "request to claim-service failed", and since we're not even consistent with respect to other services, I would suggest we remove this pattern and just let it fail (as with everything else)?